core/templates/dev/head/components/fuzzy_rule_editor_directive.html

```diff
@@ -0,0 +1,2 @@
+<input type="hidden"/>
+<button type="button" class="btn btn-danger" ng-click="removeEntry()">Remove this input</button>
```

Comment:
Hi @anmolshkl, this `ng-click` is not getting fired when user clicks on button. I have written code correctly but I am unable to debug this. Can someone who has better knowledge of angularjs can help me out?